presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoRpmControlUtils.java

```diff
@@ -0,0 +1,249 @@
+            catch (PrestoManagerException e) {
+                LOGGER.error(e.getMessage(), e.getCause());
+            }
+        }).start();
+        return Response.status(ACCEPTED).entity("Presto is being started. " +
+                "Please check back later using status api to verify that Presto has successfully started.").build();
+    }
+
+    public static Response stopUsingRpm(StopType stopType)
+    {
+        try {
+            if (!isInstalled()) {
+                LOGGER.error("Presto is not installed");
+                return Response.status(NOT_FOUND).build();
+            }
+            if (!isRunning()) {
+                LOGGER.info("Presto is not running");
+                return Response.status(OK).build();
+            }
+            switch (stopType) {
+                case TERMINATE:
+                    terminatePresto();
+                    break;
+                case KILL:
+                    killPresto();
+                    break;
+                case GRACEFUL:
+                    if (isCoordinator()) {
+                        LOGGER.error("Coordinator can't be gracefully stopped.");
+                        return Response.status(CONFLICT).build();
+                    }
+                    gracefulStop();
+                    break;
+                default:
+                    LOGGER.error("Invalid stop type: {}", stopType);
+                    return Response.status(INTERNAL_SERVER_ERROR).build();
+            }
+        }
+        catch (PrestoManagerException e) {
+            LOGGER.error(e.getMessage(), e.getCause());
+            return Response.status(INTERNAL_SERVER_ERROR).build();
+        }
+        return Response.status(OK).build();
+    }
+
+    public static Response restartUsingRpm()
+    {
+        try {
+            if (!isInstalled()) {
+                LOGGER.error("Presto is not installed");
+                return Response.status(NOT_FOUND).build();
+            }
+            int prestoRestart = executeCommand(SUBPROCESS_TIMEOUT, "service", "presto", "restart");
+            if (prestoRestart != 0) {
+                throw new PrestoManagerException("Failed to restart presto", prestoRestart);
+            }
+        }
+        catch (PrestoManagerException e) {
+            LOGGER.error(e.getMessage(), e.getCause());
+            return Response.status(INTERNAL_SERVER_ERROR).build();
+        }
+        return Response.status(OK).build();
+    }
+
+    public static Response statusUsingRpm()
+    {
+        try {
+            if (!isInstalled()) {
+                LOGGER.info("Presto is not installed");
+                return Response.status(OK).entity(GSON.toJson("Presto is not installed"))
+                        .type(APPLICATION_JSON).build();
+            }
+        }
+        catch (PrestoManagerException e) {
+            LOGGER.error(e.getMessage(), e.getCause());
+            return Response.status(INTERNAL_SERVER_ERROR).build();
+        }
+        try {
```

Comment:
what information does this provide?  Can you put a comment at the top of the method explaining?